website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -123,6 +124,7 @@ optional variables can be found in the [variables section](#orca-variables).
 - orca_default_bucket
 - db_user_password
 - db_host_endpoint
+- orca_restore_retrieval_type
```

Comment:
This is not a required variable. It should be an optional variable and we should default to standard.

Reply:
updated